english/staypuft/ghostbusters/where-is-my-proton-pack.md

```diff
@@ -10,9 +10,9 @@ Just out of reach, you see a discarded proton pack.
 
 You try to pull yourself out of the sticky mass, but with a sinking feeling you realise you are stuck.
 
-You look up in time to see a taxi cab skidding toward you through the gloopy guck. You have seconds to act
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.